references/organization/Project_story_outline.md

```diff
@@ -0,0 +1,47 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.2'
+      jupytext_version: 1.3.4
+  kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+---
+
+# <ins>Story</ins>
+
+
+
+### Part 1: Selection on observables
+
+We show that an outcome model, without a data generation model, is not enough when it comes to making causal inferences. Inferences from an outcome model alone assume a very specific data-generation process (namely, one where all the covariates are independent), and it is important to make that clear for transparency. 
```

Comment:
```suggestion
We show that an outcome model, without a proper DAG outlining the data generating process, is not enough when it comes to making causal inferences. Inferences from an outcome model alone assume a data-generation process  where all covariates are independent and might not reflect the natural data generation process.
```

Reply:
Also, I was a little confused by this part of your second sentence: "and it is important to make that clear for transparency." 

I have it removed just because I didn't know how to blend it in with my suggestions. Feel free to insert it back in a way that flows.